src/utilities/components.tsx

```diff
@@ -63,6 +63,32 @@ export function elementChildren<T extends React.ReactElement<{}>>(
   ) as T[];
 }
 
+interface ConditionalWrapperProps {
+  children: any;
+  condition: boolean;
+  wrapper: (children: any) => any;
+}
+
+export function ConditionalWrapper({
```

Comment:
Added a couple helper components for conditionally rendering blocks and conditionally wrapping blocks